packages/legend-studio/src/stores/editor-state/element-editor-state/mapping/PureInstanceSetImplementationState.ts

```diff
@@ -217,4 +320,28 @@ export class PureInstanceSetImplementationState extends InstanceSetImplementatio
       }
     }
   }
+
+  *convertFilterMapping(): GeneratorFn<void> {
```

Comment:
This is probably not the right name, it should just be `convertFilter`, `filterMapping` implies something else